renormalizer/model/mlist.py

```diff
@@ -14,8 +14,8 @@
 
 class MolList:
 
-    def __init__(self, mol_list: List[Mol], j_matrix: Union[Quantity, np.ndarray, None], scheme: int=2, period=False):
-        self.period = period
+    def __init__(self, mol_list: List[Mol], j_matrix: Union[Quantity, np.ndarray, None], scheme: int = 2, periodic: bool = False):
+        self.period = periodic
```

Comment:
It is better to replace all the `period` with `periodic`.

Reply:
I agree. But I think @jiangtong1000 may have some code that uses this property. @jiangtong1000 are you okay with this modification?

Reply:
SURE